frontend/simple/views/DesignSystem.vue

```diff
@@ -1,87 +1,160 @@
+        </tr>
+        <tr>
+          <td class="is-narrow"><code>$text-light<br />.has-text-text-light</code></td>
+          <td class="has-text-text-light">Use this color on non critical texts.</td>
+        </tr>
+        <tr>
+          <td class="is-narrow"><code>$link</code></td>
+          <td>Make sure <a href="#">the link is readable</a> for everyone, this is, accessible.</td>
+        </tr>
+      </table>
+
+      <h2 class="title is-3 c-title">Spacing</h2>
+      <table class="table is-fullwidth is-vertical-middle c-table">
+        <thead>
+          <th>code</th>
+          <th>demo</th>
+        </thead>
+        <tr v-for="spacer in ['xs', 'sm', '', 'lg']">
+          <td class="is-narrow" v-if="spacer"><code>$gi-spacer-{{spacer}}</code></td>
+          <td class="is-narrow" v-else="spacer"><code>$gi-spacer</code></td>
+          <td><span class="c-spacer" :class="spacer"></span></td>
+        </tr>
+      </table>
+
+      <h2 class="title is-3 c-title">Color</h2>
+
+      <p class="is-size-5">Bulma has much <a href="https://bulma.io/documentation/customize/variables/">more variables</a> that we haven't needed to use so far. Stick with Brand and Support variables and avoid raw color variables ($white, $blue, $grey, etc...) so when we have multiple themes, the migration between those is impartial from raw variables. (Note: This is still happening around on borders color and should be fixed)</p>
 
-    </div>
-  </div>
+      <br/><br/>
+
+      <h3 class="title is-4">Brand</h3>
+      <table class="table is-fullwidth is-vertical-middle c-table">
+        <thead>
+          <th>color</th>
+          <th>code</th>
+          <th>usage</th>
+        </thead>
+        <tr v-for="color in config.colors.brand">
+          <td class="is-narrow"><span class="c-palette" :class="`has-background-${color.name}`"></span></td>
+          <td class="is-narrow"><code>${{color.name}} <template v-if="!color.noClass"><br/>.has-background-{{color.name}}</template></code></td>
+          <td v-html="color.usage"></td>
+        </tr>
+      </table>
+
+      <h3 class="title is-4">Support</h3>
+      <table class="table is-fullwidth is-vertical-middle c-table">
+        <thead>
+          <th>color</th>
+          <th>code</th>
+          <th>usage</th>
+        </thead>
+        <tr v-for="color in config.colors.support">
+          <td class="is-narrow"><span class="c-palette" :class="`has-background-${color.name}`"></span></td>
+          <td class="is-narrow"><code>${{color.name}} <template v-if="!color.noClass"><br/>.has-background-{{color.name}}</template></code></td>
+          <td v-html="color.usage"></td>
```

Comment:
See [v-html concern](https://github.com/okTurtles/group-income-simple/pull/493/files#r234470204)

Reply:
Here this is not a concern because it is not "user-provided content"

> Only use v-html on trusted content and never on user-provided content.